CMakeLists.txt

```diff
@@ -80,10 +80,10 @@ if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-command-line-argument")
 else()
   set(FP_MODE_FLAG "-frounding-math -ftrapping-math")
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wmissing-format-attribute -Wsuggest-final-methods -Wsuggest-final-types -Wvolatile -Werror")
```

Comment:
Why remove -Werror?

Reply:
I made it global. Previously it was limited to GCC and not Clang. I'm currently removing build errors.

It doesn't build in my environment right now, probably because Git silently broke it during an otherwise conflict-free merge of `devel` into this PR's branch.